### PR TITLE
feat: feed settings filters

### PR DESCRIPTION
**Pull request by capJavert**
## Changes

- API changes pending (validation) https://github.com/dailydotdev/daily-api/pull/2546

image

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://as-814-feed-settings-filters.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/feeds/FeedSettings/sections/FeedSettingsFiltersSection.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+            label="Min views"
+            placeholder="No limit"
+            required
+            min={0}
+            max={1000}
+            valueChanged={(value) => setData({ minViews: +value })}
+          />
+        </div>
+      </div>
+      <div className="flex flex-col gap-4">
```
Comment (sshanzel) [inline packages/shared/src/components/feeds/FeedSettings/sections/FeedSettingsFiltersSection.tsx:119]: I think we shouldn't need the gap here as it is the only child. Maybe even the wrapper 👀

File: packages/shared/src/components/feeds/FeedSettings/sections/FeedSettingsFiltersSection.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+          </Typography>
+        </div>
+        <div className="flex gap-4">
+          <TextField
+            className={{
+              container: 'w-full tablet:max-w-70',
+            }}
+            leftIcon={<UpvoteIcon />}
+            defaultValue={feed.flags?.minUpvotes}
+            name="name"
```
Comment (sshanzel) [inline packages/shared/src/components/feeds/FeedSettings/sections/FeedSettingsFiltersSection.tsx:91]: I think both text fields have the same name.

File: packages/shared/src/components/feeds/FeedSettings/sections/FeedSettingsFiltersSection.tsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+          </Typography>
+          <Typography
+            type={TypographyType.Callout}
+            color={TypographyColor.Tertiary}
+          >
+            Include only posts published within a specific time range.
+          </Typography>
+        </div>
+        <Radio
+          name="theme"
```
Comment (sshanzel) [inline packages/shared/src/components/feeds/FeedSettings/sections/FeedSettingsFiltersSection.tsx:63]:
```suggestion
          name="feedRangeFilter"
```

**Review by sshanzel — COMMENTED** [review]
Just the name having the same value, other than that, it looks amazing 🚀

File: packages/shared/src/components/feeds/FeedSettings/sections/FeedSettingsFiltersSection.tsx (hunk 2 above)
Comment (capJavert) [reply packages/shared/src/components/feeds/FeedSettings/sections/FeedSettingsFiltersSection.tsx:91]: Good catch!

File: packages/shared/src/components/feeds/FeedSettings/sections/FeedSettingsFiltersSection.tsx (hunk 1 above)
Comment (capJavert) [reply packages/shared/src/components/feeds/FeedSettings/sections/FeedSettingsFiltersSection.tsx:119]: yes!

**Pull request merged**